### ISSUE TITLE
Add basic cookie based multiple submission prevention.

### ISSUE THREAD

**Issue by russleyshaw** [siggame/registre#8]

**Comment by renodubois** [comment]
In terms of handling multiple submissions, I think it might be better to just allow them, and we can trim the entries at the end of the submission period. (likely by email)

I can imagine a scenario where someone enters something wrong on accident and has no real way to edit it, so submitting a second time makes sense. However, there are definitely going to be some issues with allowing multiple submissions as well, but I'm not sure what trade off we want to make there.

**Comment by russleyshaw** [comment]
I agree. I am keeping this issue up though so that it can be added as a minor improvement later.
